app/frontend/src/cljs/teet/comments/comments_controller.cljs

```diff
@@ -58,9 +72,57 @@
 
 
   DeleteComment
-  (process-event [{comment-id :comment-id} app]
+  (process-event [{:keys [comment-id commented-entity]} app]
     (t/fx app
           {:tuck.effect/type :command!
            :command          :comment/delete-comment
            :payload          {:comment-id comment-id}
-           :result-event     common-controller/->Refresh})))
+           :success-message (tr [:notifications :comment-deleted])
+           :result-event     (partial ->DeleteCommentResult commented-entity)}))
+
+  DeleteCommentResult
+  (process-event [{:keys [commented-entity]} app]
+    (t/fx app
+          (comments-query commented-entity)))
+
+  OpenEditCommentDialog
+  (process-event [{:keys [comment-id commented-entity comment]} app]
+    (-> app
+        (assoc-in [:stepper :dialog] {:type :edit-comment})
```

Comment:
Not sure if this stepper should be used as a place for "not stepper" related dialogs? Or it should be renamed, the whole :stepper is from a time when the project navigator was supposed to be a lot different.

Reply:
Given how ubiquitous dialogs are, I'd like to have an easy, general mechanism for firing up dialogs. So I'd go with renaming.